README.md

```diff
@@ -56,14 +53,12 @@ Kresus is an open-source libre self-hosted personal finance manager. It allows y
   * [ ] Improve log file and add logrotate
   * [ ] Add user who will access the app (by default every one has access to the installed app)
 
-## Links
+## Documentation and resources
 
- * Report a bug about this package: https://github.com/YunoHost-Apps/kresus_ynh
- * Report a bug about Kresus itself: https://framagit.org/kresusapp/kresus
- * Kresus website: https://kresus.org
- * YunoHost website: https://yunohost.org
-
----
+* Official app website: https://framagit.org/kresusapp/kresus
+* Upstream app code repository: https://framagit.org/kresusapp/kresus
+* YunoHost documentation for this app: https://yunohost.org/app_kresus
+* Report a bug: https://github.com/YunoHost-Apps/kresus_ynh/issues
```

Comment:
Can we add back `* Official documentation: https://kresus.org/en/doc.html` please?

Reply:
Added ;)